src/date.ml

```diff
@@ -584,8 +584,8 @@ value string_slash_of_date conf date =
         " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 1) ^ ")"
   | Dgreg d Dfrench ->
       let d1 = Calendar.french_of_gregorian d in
-      slashify_dmy (translate_dmy conf (decode_dmy conf d1) Dfrench True) d1 ^
-        " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 2) ^ ")"
+      slashify_dmy (translate_dmy conf (decode_dmy conf d1) Dfrench True) d1(* ^
+        " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 2) ^ ")" *)
```

Comment:
Slashify are minimised version of dates that are used in table where we can't afford to print (French Republican) on its column, we need space for other col. + the short republican date like PL/21/IX speaks for itself > see #450. 

So yes we can probably remove (Hebrew) from hebraic short dates it is also very explicit format with specific month abreviations. (that said I see a bug on those dates using hebrew calendar: we can't click on them to go to the calendar tool!)

What is sure is that we can't remove the info on Julian dates that looks too much like Gregorian ones!